src/distilabel/steps/task/generate_embeddings.py

```diff
@@ -45,6 +45,8 @@ class GenerateEmbeddings(Step):
     llm: LLM
 
     def load(self) -> None:
+        """Loads the `LLM` used to generate the embeddings."""
+        super().load()
```

Comment:
If all the steps inheriting from `Step` need to actually call the `super().load()` isn't it better if we just add it as a separate method that runs within the `post_init`? i.e. `model_post_init` on `_Step`?

Reply:
Sadly we can't do that :( We need to create the logger after having called `setup_logging` in each process.

Reply:
Can we then just do `step.load()` and then `step.setup_logging`? But to manage that within the `Pipeline` instead of having to always add that as part of the `load` method :/